java/test/src/main/java/io/ray/test/CrossLanguageInvocationTest.java

```diff
@@ -188,6 +188,91 @@ public void testPassActorHandleFromJavaToPython() {
     Assert.assertEquals(res.get(), "3".getBytes());
   }
 
+  @Test
+  public void testExceptionSerialization() throws InvalidProtocolBufferException {
```

Comment:
Can you move these tests to a separate `CrossLanguageExceptionTest`? This test is already a bit too long

Reply:
Cross language test needs a lot of helper code, for example, Python file, Java class, Java method... So, it will be simpler to make the tests together. And, it will be faster to test all cross language cases in one cluster.